database/migrations/2016_11_28_151416_create_vouchers_table.php

```diff
@@ -0,0 +1,34 @@
+<?php
+
+use Illuminate\Support\Facades\Schema;
+use Illuminate\Database\Schema\Blueprint;
+use Illuminate\Database\Migrations\Migration;
+
+class CreateVoucherstable extends Migration
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::create('vouchers', function (Blueprint $table) {
+            $table->increments('id');
+            $table->string('name',100)->unique();
+            $table->integer('poin');
+            $table->string('value',100);
```

Comment:
I have a question: please explain for me about value meaning, why do you using string there?